api/routes/index.js

```diff
@@ -1,12 +1,14 @@
 const express = require("express");
 const router = express.Router();
 
-const itemsRoutes = require('./itemsRoutes');
+const itemsRoutes = require("./itemsRoutes");
 const signUpRouter = require("./signUpRouter");
 const signInRouter = require("./signInRouter");
+const itemsFetchRouter = require("./itemsFetchRouter");
 
-router.use('/items', itemsRoutes.router);
+router.use("/items", itemsRoutes.router);
 router.use("/signup", signUpRouter.router);
 router.use("/signin", signInRouter.router);
+router.use("/itemsFetch", itemsFetchRouter.router);
```

Comment:
itemsFetch 와 items는 하나로 합져 주면 좋을 것 같습니다.
signup, signin은 조금 특수한 경우긴 한데, items 같은 경우 확실하게 어떤 자원을 사용하는지 나타내는 것이 좋을 것 같아요.
- itme detail : ip:port/items/:itemId
- item list : ip:port/items

```suggestion
router.use("/items", itemsRoutes.router);
router.use("/signup", signUpRouter.router);
router.use("/signin", signInRouter.router);
```

Reply:
합쳐보았습니다!